task_DEV_2/task_DEV_2/Program.cs

```diff
@@ -6,48 +6,65 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Enter the size of the array :");
+            Console.WriteLine("Enter the length of the array :");
             int arrayLength = Convert.ToInt32(Console.ReadLine());
 
             if (arrayLength <= 20)
             {
-                int[] inputNumbers = new int[arrayLength];
+                int[] inputNumbers = CreateNumbersArray(arrayLength);
+                PrintNumberArraryToConsole(inputNumbers, arrayLength);
+                SwapMinMaxElements(inputNumbers, arrayLength);
```

Comment:
After swapping would be more logical do smth like this
inputNumbers = SwapMinMaxElements(inputNumbers, arrayLength);

Reply:
done